lightning/src/events/mod.rs

```diff
@@ -1670,6 +1671,7 @@ impl Writeable for Event {
 					(2, payment_hash, option),
 					(4, path.hops, required_vec),
 					(6, path.blinded_tail, option),
+					(8, path.trampoline_hops, optional_vec),
```

Comment:
I am trying to understand how types get assigned in LDK - it's an optional field here so shouldn't it have an odd type value ? thanks for clarifying.

Reply:
thank you I understand now.